.changes/next-release/enhancement-AWSRequest-24612.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
While we generally want Changelog entries, `AWSRequest` isn't a public interface and is omitted from our public docs. We should probably keep this out of the Changelog to avoid any confusion.

Reply:
Ah got it I thought it was needed for everything! I'll remove it.